src/backend/executor/execSRF.c

```diff
@@ -236,6 +236,9 @@ ExecMakeTableFunctionResult(SetExprState *setexpr,
 			fcinfo->isnull = false;
 			rsinfo.isDone = ExprSingleResult;
 			result = FunctionCallInvoke(fcinfo);
+			setexpr->isSquelchSupported = rsinfo.returnMode & SFRM_Squelch;
+			/* Reset SFRM_Squelch bit */
+			rsinfo.returnMode &= ~SFRM_Squelch;
```

Comment:
Why do we need to reset the `SFRM_Squelch` flag?

Reply:
Further in line 254 there is a string comparison 
`if (rsinfo.returnMode == SFRM_ValuePerCall)`
which otherwise affected by extra SFRM_Squelch flag. From the other side we don't need it anymore, so just reset.